### PR TITLE
Added hourly rates and blocks/chest to powder grinding tracker

### PR DESCRIPTION
**Pull request by KingsMMA**

### CODE REVIEW

**Review by hannibal002 — APPROVED** [review]
image
I don't like this feature, it shows how bad I am.

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java [hunk 1]
```diff
@@ -719,15 +719,33 @@ public class Mining {
...
+		name = "Pause Timer",
+		desc = "How many seconds does it wait before pausing"
+	)
+	@ConfigAccordionId(id = 9)
+	@ConfigEditorSlider(
+		minValue = 1,
+		maxValue = 20,
+		minStep = 1
+	)
+	public int powderGrindingPauseTimer = 3;
```
[the remark this answers is not in the document]
Comment (KingsMMA) [reply src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java:747]: I can increase the default setting if you want; I used 10 when testing as it gave me time to open chests

[the remark this answers is not in the document]
Comment (NopoTheGamer) [reply src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java:747]: eh ig?

**Comment by Ascynx** [comment]
> image
> I don't like this feature, it shows how bad I am. 

Same, same 😭

**Review by hannibal002 — APPROVED** [review]

**Review by hannibal002 — CHANGES_REQUESTED** [review]
image

Is this my fault? I've started the version again after one day and then some values are negative?

**Comment by KingsMMA** [comment]
> image
> 
> Is this my fault? I've started the version again after one day and then some values are negative?

I've updated saving and loading for the elapsed and blocks mined values.
Do you have multiple tools with compact?
The blocks mined is updated based off of your held tools compact counter, which may have caused the negative values

**Comment by hannibal002** [comment]
@KingsMMA sadly yes. You have to compare the compact value AND the item UUID as well. :/

**Comment by hannibal002** [comment]
Since no more edits for almost 2 weeks, I move this PR into drafts. @KingsMMA Feel free to open the PR as soon as you think it's ready :)

**Pull request closed without merging**